scripts/make_release_notes.py

```diff
@@ -244,11 +252,11 @@ def read_changelog_section(filename, single_version=None):
     # Discard all lines until we see a heading that either has the version the
     # user asked for or any version.
     if single_version:
-      initial_heading = re.compile(r'^#.*%s' % re.escape(single_version))
+      initial_heading = re.compile(r'^# .*%s' % re.escape(single_version))
```

Comment:
Should these also take the form `#{1,6}`?

Reply:
Done.